0-0-admin-tasks.md

```diff
@@ -2,6 +2,8 @@
 layout: page
 category: walkthrough
 title: Administration tasks
+contact-admin: email gsi@oicr.on.ca or file a JIRA ticket in _GSI Common_
```

Comment:
Could move to config so that all organization-specific config is in one place

Reply:
I like that. Fixed.